index.js

```diff
@@ -2,17 +2,44 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+// added defer to the script tag 
 
 /***** Deliverable 2 *****/
+const newHeader = document.querySelector('#header')
 
+newHeader.style.color = 'red'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const firstNewImg = document.querySelector("#profile img")
+firstNewImg.src = traveler.photo
+
+firstNewImg.alt = traveler.name 
+
+console.log(firstNewImg)
+
+const travelerName = document.querySelector("#profile > h2")
+travelerName.innerText = traveler.name 
```

Comment:
Good~! You can also use `textContent` instead of `innerText`. The main difference is that `textContent` doesn't consider the CSS whereas `innerText` does. [Take a look at this stackoverflow post about it](https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext)